src/ogma/features/nodeGrouping.ts

```diff
@@ -0,0 +1,205 @@
+import {StyleRule, Transformation, Node, NodeList} from '@linkurious/ogma';
+import {ConflictValue, LkEdgeData, LkNodeData, MissingValue} from '@linkurious/rest-client';
+
+import {LKOgma} from '../index';
+import {Tools} from '../../tools/tools';
+
+export const LKE_NODE_GROUPING_EDGE = 'LKE_NODE_GROUPING_EDGE';
+
+export class NodeGroupingTransformation {
+  public transformation?: Transformation<LkNodeData, LkEdgeData>;
```

Comment:
why public?

Reply:
I will need to access it from the frontend